api/core/domain/task_variant.py

```diff
@@ -40,6 +41,7 @@ class SerializableTaskVariant(BaseModel):
     is_public: bool | None = None
 
     creation_chat_messages: list[ChatMessage] | None = None
+    used_integration_kind: IntegrationKind | None = None
```

Comment:
You don't fill that value correct ?

Reply:
I do:
`# Register the integration used for the agent, so we can provide pertinent answers in the playground chat
                await self.storage.task_variants.update_task(
                    relevant_run_and_agent.agent.task_id,
                    used_integration_kind=integration_slug,
                )`